src/frontend/src/api/ApiClient.ts

```diff
@@ -16,16 +23,109 @@ export default class ApiClient {
 
     async getContractsAsync(projectId: string) {
         const url = this.resourceCollection.contracts(projectId);
-        return this.httpClient.getAsync<ApiCollection<Contract>, FusionApiHttpErrorResponse>(url);
+        const response = await this.httpClient.getAsync<
+            ApiCollection<Contract>,
+            FusionApiHttpErrorResponse
+        >(url);
+        return response.data.value;
     }
 
     async getAvailableContractsAsync(projectId: string) {
         const url = this.resourceCollection.contracts(projectId);
-        return this.httpClient.getAsync<ApiCollection<AvailableContract>, FusionApiHttpErrorResponse>(url);
+        const response = await this.httpClient.getAsync<
+            ApiCollection<AvailableContract>,
+            FusionApiHttpErrorResponse
+        >(url);
+        return response.data.value;
     }
 
     async getPersonnelAsync(projectId: string, contractId: string) {
         const url = this.resourceCollection.personnel(projectId, contractId);
         return this.httpClient.getAsync<ApiCollection<Personnel>, FusionApiHttpErrorResponse>(url);
     }
+
+    async createContractAsync(projectId: string, contract: Contract) {
+        const url = this.resourceCollection.contracts(projectId);
+        const response = await this.httpClient.postAsync<
+            Contract,
+            Contract,
+            FusionApiHttpErrorResponse
+        >(url, contract);
+        return response.data;
+    }
+
+    async updateContractAsync(projectId: string, contractId: string, contract: Contract) {
+        const url = this.resourceCollection.contract(projectId, contractId);
+        const response = await this.httpClient.putAsync<
+            Contract,
+            Contract,
+            FusionApiHttpErrorResponse
+        >(url, contract);
+
+        return response.data;
+    }
+
+    async createExternalCompanyReprasentiveAsync(
```

Comment:
createExternalCompanyReprasentiveAsync should probably be createExternalCompanyRepresentativeAsync ?

Reply:
@olerichard fixed